src/plugin.js

```diff
@@ -183,30 +209,31 @@ export default class TagWrangler extends Plugin {
             search = searchPlugin && searchPlugin.instance,
             query = search && search.getGlobalSearchQuery(),
             random = this.app.plugins.plugins["smart-random-note"],
-            menu = e.obsidian_contextmenu.addItem(item("pencil", "Rename #"+tagName, () => this.rename(tagName)));
+            menu = e.obsidian_contextmenu.addItem(item("pencil", `${this.it.Rename} #`+tagName, () => this.rename(tagName)));
 
         menu.addSeparator();
         if (tagPage) {
             menu.addItem(
-                item("popup-open", "Open tag page", (e) => this.openTagPage(tagPage, false, Keymap.isModEvent(e)))
+                item("popup-open", this.it[OPEN_TAG_PAGE], (e) => this.openTagPage(tagPage, false, Keymap.isModEvent(e)))
             )
         } else {
             menu.addItem(
-                item("create-new", "Create tag page", (e) => this.createTagPage(tagName, Keymap.isModEvent(e)))
+                item("create-new",this.it[CREATE_A_TAG], (e) => this.createTagPage(tagName, Keymap.isModEvent(e)))
             )
         }
 
         if (search) {
             menu.addSeparator().addItem(
-                item("magnifying-glass", "New search for #"+tagName, () => search.openGlobalSearch("tag:" + tagName))
+                item("magnifying-glass", `${this.it[NEW_SEARCH_FOR]} #`+tagName, () => search.openGlobalSearch("tag:" + tagName))
             );
             if (query) {
                 menu.addItem(
-                    item("sheets-in-box", "Require #"+tagName+" in search"  , () => search.openGlobalSearch(query+" tag:"  + tagName))
+                    item("sheets-in-box", `${this.it[REQUIRE]} #${tagName} ${this.it[IN_SEARCH]}`, 
+                    () => search.openGlobalSearch(query+" tag:"  + tagName))
                 );
             }
             menu.addItem(
-                item("crossed-star" , "Exclude #"+tagName+" from search", () => search.openGlobalSearch(query+" -tag:" + tagName))
+                item("crossed-star" , `${this.it[EXCLUDE]} #${tagName} ${this.it[FROM_SEARCH]}`, () => search.openGlobalSearch(query+" -tag:" + tagName))
```

Comment:
Concatenating strings violates the very basics of translation design.   Concatenation fails for nearly all languages that not the source language.  One must NEVER concatenate strings, as word order changes, conjugations dependent on variables, etc.    This will fail for languages that require the verb after the object (from-search #tag exclude), languages that require the verb after the complement (from-search exclude #tag), where conjugations are required on the tag-name (exclude #tag-name":ssa" from-search), etc.